### PR TITLE
Group fix adhikram

### PR DESCRIPTION
**Pull request by Adhikram**

### CODE REVIEW

**Review by kaydee0502 — APPROVED** [review]

File: devsnest/app/models/group_member.rb [hunk 1]
```diff
@@ -5,6 +5,18 @@ class GroupMember < ApplicationRecord
   belongs_to :group
   after_create :send_all_steps_completed_mail
   after_create :set_prevoiusly_joined_a_group
+  after_create :increase_members_count
```
Comment (kaydee0502) [inline devsnest/app/models/group_member.rb:8]: `after_commit :adjust_member_count,on: [:create, :destroy]`

File: devsnest/app/models/group_member.rb [hunk 2]
```diff
@@ -5,6 +5,18 @@ class GroupMember < ApplicationRecord
...
+  before_destroy :decrese_members_count
+
+  def decrese_members_count
+    group = Group.find(group_id)
+    group.update(members_count: group.members_count - 1)
+  end
+
```
Comment (kaydee0502) [inline devsnest/app/models/group_member.rb:15]:
```
  def adjust_member_count
    self.group.update(members_count: self.group.group_members.count)
  end
```

Comment (Adhikram) [reply devsnest/app/models/group_member.rb]: done

File: devsnest/app/models/group_member.rb (hunk 1 above)
Comment (Adhikram) [reply devsnest/app/models/group_member.rb]: Done

**Pull request merged**